realtime/server.js

```diff
@@ -21,14 +21,14 @@ var people = {};
 var usernames = {};
 var messages_typing = {};
 
-function getOptions(form, target, method) {
+function getOptions(form, params, method) {
```

Comment:
This is the real-time server. For the real-time server, it's just part of the URL.